playbooks/roles/postgresql_aggregation/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+- name: Install packages
+  apt:
+    name: "{{ item }}"
+    install_recommends: yes
+    state: present
+  with_items: postgresql_packages
+
+- name: install requirements for role
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items: postgresql_aggregation_requirements
+
+- name: check if user already exists
```

Comment:
Is this task part of the workaround for https://github.com/ansible/ansible-modules-core/issues/297?